app/blog/20161004-how-to-update-wordpress.static.md

```diff
@@ -0,0 +1,53 @@
+---
+title: How to update WordPress
```

Comment:
Perhaps update the title to reflect this content is specific to updating the Divi and accompanying n.io child theme as opposed to a core WordPress code update